rtengine/ipdehaze.cc

```diff
@@ -229,100 +269,143 @@ void ImProcFunctions::dehaze(Imagefloat *img)
 
     int patchsize = max(int(5 / scale), 2);
     float ambient[3];
-    array2D<float> &t_tilde = dark;
     float max_t = 0.f;
 
     {
-        int npatches = 0;
         array2D<float> R(W, H);
         array2D<float> G(W, H);
         array2D<float> B(W, H);
         extract_channels(img, R, G, B, patchsize, 1e-1, multiThread);
-    
-        patchsize = max(max(W, H) / 600, 2);
-        npatches = get_dark_channel(R, G, B, dark, patchsize, nullptr, false, multiThread);
-        DEBUG_DUMP(dark);
 
-        max_t = estimate_ambient_light(R, G, B, dark, patchsize, npatches, ambient);
+        {
+            constexpr int sizecap = 200;
+            float r = float(W)/float(H);
```

Comment:
Can't see any modification of `r` in the diff below. If so, it should read

```c++
const float r = static_cast<float>(W) / static_cast<float>(H);
```

Reply:
Fixed